db/db_iter.cc

```diff
@@ -1280,15 +1279,15 @@ void DBIter::SeekToLast() {
     if (!Valid()) {
       return;
     } else if (user_comparator_->Equal(*iterate_upper_bound_, key())) {
-      Prev();
+      ReleaseTempPinnedData();
+      PrevInternal();
```

Comment:
While I agree that the previous code might be double decrementing in this particular case, I am a little concerned that by replacing `Prev` with `PrevInternal` here, we are skipping some of the logic in `Prev`, like, setting the direction to backwards in `ReverseToBackward`. I am not sure about the implications of that. 

But I guess I am ok with this as long as all our tests pass.

Reply:
All tests passed, so this might not be a concern. We can proceed to landing this diff.